user-guide/Advanced_Modules/Router_Control/Creating_and_customizing_RC_configs/Designing_a_matrix_tab_page_in_the_Router_Control_module.md

```diff
@@ -72,7 +72,7 @@ In order to add such a matrix, do the following:
 
 1. Add a matrix like in the procedure above, until you get to the *Add Matrix* dialog box.
```

Comment:
1. Add a matrix like in the procedure above, until you get to the *Advanced configuration* section.
2.  Open the *Advanced configuration* section
3. Enter the following syntax in the *Custom matrix* box: